packages/mobile/src/components/audio/GoogleCast.tsx

```diff
@@ -135,7 +137,18 @@ export const useChromecast = () => {
     }
   }, [client, seek])
 
+  useEffect(() => {
+    if (
+      castState === CastState.CONNECTED ||
+      castState === CastState.CONNECTING
+    ) {
+      TrackPlayer.setVolume(0)
+    } else {
+      TrackPlayer.setVolume(1)
```

Comment:
do the volume controls control the device you're casting to?

Reply:
they do on android, but don't on iOS (see PR description).
this `setVolume` controls the phone though, and the way chromecast works, both the phone and the cast device are playing the track at the same time. it doesn't have to be that way, but would mean we have to rewrite a lot of the player control logic